.github/workflows/benchmark.yml

```
@@ -33,15 +33,12 @@ jobs:
         uses: actions/checkout@v2
         if: github.event_name == 'workflow_dispatch'
 
+# TODO(#XXX): `rustup show` installs the rustc version specified in the `rust-toolchain` file
+# according to https://rust-lang.github.io/rustup/overrides.html. We don't use actions-rs due to the
+# lack of support of `rust-toolchain` files with TOML syntax:
+# https://github.com/actions-rs/toolchain/issues/126.
```

Can you please fill in the XXX with the issue for this, and add this to the other two places where these changes are made? Oh, also I forgot to add indentation to the comment.

```suggestion
      # TODO(#XXX): `rustup show` installs the rustc version specified in the `rust-toolchain` file
      # according to https://rust-lang.github.io/rustup/overrides.html. We don't use actions-rs due
      # to the lack of support of `rust-toolchain` files with TOML syntax:
      # https://github.com/actions-rs/toolchain/issues/126.
```

---

Oh I missed that :sweat:

---

Should we create issue on our git repo?

---

No worries. Yeah, just so we don't forget about removing the workaround when we get the chance.